cypress/support/fragments/data_import/searchInventory.js

```diff
@@ -37,19 +37,17 @@ const getInstanceHRID = () => {
             },
           })
             .then(({ body: { relatedInstanceInfo } }) => {
-              return relatedInstanceInfo.hridList[0];
+              return relatedInstanceInfo.hridList;
             });
         });
     });
 };
 
 const searchInstanceByHRID = (id) => {
-  cy.get('#input-inventory-search-qindex').select('Instance HRID');
-  cy.do([
-    // Select({ id: 'input-inventory-search-qindex' }).choose('Instance HRID'),
-    TextField({ id: 'input-inventory-search' }).fillIn(id),
-    Button('Search').click()
-  ]);
+  InventoryInstances.waitContentLoading();
+  cy.do(Select({ id: 'input-inventory-search-qindex' }).choose('Instance HRID'));
+  cy.do(TextField({ id: 'input-inventory-search' }).fillIn(id));
+  cy.do(Button('Search').click());
```

Comment:
You can rewrite here:
cy.do(
Select({ id: 'input-inventory-search-qindex' }).choose('Instance HRID');
TextField({ id: 'input-inventory-search' }).fillIn(id)
Button('Search').click())